docs/src/main/sphinx/connector/delta-lake.md

```diff
@@ -780,7 +781,34 @@ directly or used in conditional statements.
 - `$file_size`
   : Size of the file for this row.
 
+### Replace tables
+
+The connector supports replacing an existing table, as
+an atomic operation. Atomic table replacement creates a new snapshot with the
+new table definition as part of the [table history](#delta-lake-history-table).
+
+To replace a table, use [`CREATE OR REPLACE TABLE`](/sql/create-table) or
+[`CREATE OR REPLACE AS`](/sql/create-table-as).
+
+In this example, a table `example_table` can be replaced by a completely new
+definition:
+
+```sql
+CREATE TABLE example_table (
+    a BIGINT,
+    b DATE,
+    c BIGINT)
+WITH (partitioning = "partitioned_by");
```

Comment:
that has to be a column of the table or a function to process the table. See other example

Reply:
@ebyhr?

Reply:
Either same as other example or just remove it. Having that partitioning defined does nothing to the value of the example